src/common/reader.js

```diff
@@ -1197,8 +1199,37 @@ class Reader {
 						}
 						else {
 							this._lastView.navigate({ annotationID: annotation.id });
+							// When annotation is added from "Find in" popup, close it and re-focus the view
+							this.toggleFindPopup({ primary: this._lastViewPrimary, open: false });
+							this.focusView(this._lastViewPrimary);
 						}
 					}
+					// After a small delay for focus to settle, announce to screen readers that annotation
+					// is selected and how one can manipulate it
+					setTimeout(() => {
+						let a11yAnnouncement = this._getString("pdfReader.a11yAnnotationSelected");
```

Comment:
We typically use single quotes unless double quotes are necessary.

Reply:
Gotcha, I'll keep it in mind! I re-pushed everything as it was easier to find the double quotes that way